odl/space/numpy_ntuples.py

```diff
@@ -563,7 +587,7 @@ class Fn(FnBase, Ntuples):
     Its elements are represented as instances of the `FnVector` class.
     """
 
-    def __init__(self, size, dtype, **kwargs):
+    def __init__(self, size, dtype='float64', **kwargs):
```

Comment:
Why this change? `Fn` is generic and to me there is no canonical choice of default `dtype`.